haystack/document_stores/utils.py

```diff
@@ -483,25 +482,25 @@ def elasticsearch_index_to_document_store(
         # Get content and metadata of current record
         content = record["_source"].pop(original_content_field, "")
         if content:
-            record_doc = {"content": content, "meta": {}}
+            record_doc = Document(content=content, meta={})
```

Comment:
How are ids handled here? Could it be that we want to set them explicitly just as they were before? For example, what if there were two documents with the same content but different ids in the elasticsearch index? Do we silently drop documents with the same content as duplicates because of no id/id_hash_keys being set here explicitly?